provisioning/roles/mangaki_source/templates/settings.ini.j2

```diff
@@ -3,6 +3,7 @@
 {% if mangaki_source_debug|bool %}
 [debug]
 DEBUG = True
+SHOW_REVISION = True
```

Comment:
What is `j2` supposed to mean, in `settings.ini.j2`?

Reply:
Jinja 2, it's the template language used by Ansible.